esrally/metrics.py

```diff
@@ -1314,11 +1314,13 @@ def from_dict(cls, d):
         else:
             user_tags = {}
 
-        # Don't restore a few properties like cluster because they (a) cannot be reconstructed easily without knowledge of other modules
+        team_revision = d.get("cluster").get("team-revision")
```

Comment:
If there is no `cluster` property this will fail because `#get()` will return `None`.

Reply:
thank you for  catching this!